core/embed/upymod/modtrezorcrypto/modtrezorcrypto-tropic.h

```diff
@@ -0,0 +1,174 @@
+/*
+ * This file is part of the Trezor project, https://trezor.io/
+ *
+ * Copyright (c) SatoshiLabs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#if USE_TROPIC
+
+#include <sec/secret.h>
+#include <sec/tropic_transport.h>
+#include "libtropic.h"
+
+#define PKEY_INDEX_BYTE PAIRING_KEY_SLOT_INDEX_0
```

Comment:
I think this can be removed.

Reply:
https://github.com/trezor/trezor-firmware/pull/4172/commits/07425d69411a20883084721064158b4c5a70e8e5#diff-20d5e6900b62b56f17de07c11dd9dafc13dcefc43b3b863af9a6aa6f4775b2afL26